package.json

```diff
@@ -8,57 +8,62 @@
     "build": "gulp buildOptimized",
     "testAndBuild": "gulp testAndBuildOptimized",
     "buildGroundControlSkeleton": "gulp buildGroundControlSkeleton",
-    "postinstall": "git clone -b full/4.5.3 --single-branch https://github.com/ckeditor/ckeditor-releases.git node_modules/ckeditor"
```

Comment:
Why is the `--single-branch` option removed? This would improve the clone as it fetches only the history for the requested branch